app/homeStyles.ts

```diff
@@ -1,14 +1,21 @@
 import styled from "styled-components/native";
+import { Dimensions } from "react-native";
 
-export const Container = styled.TouchableOpacity`
-  /* padding-top: 20px;
-  padding-bottom: 20px; */
-`;
+export const Container = styled.TouchableOpacity``;
 
 export const FlatListContainer = styled.View`
   height: fit-content;
 `;
 
+export const FullWidthContainer = styled.View`
+  margin: 0 -12px;
+  width: ${Dimensions.get("window").width}px;
```

Comment:
is this needed? can you not just use 100%

Reply:
auto works equally as well and has now been changed

Reply:
ok great, can you also remove the import for it if its not being used anywhere else in this file